package.json

```diff
@@ -1,42 +1,43 @@
-		"restart": "npm run stop; npm run start",
-		"status": "docker-compose exec mshots bash -c 'echo mShots Master process: `set -o pipefail; ps ax | grep Master | grep -v grep | awk \"{print \\\\$1}\" || echo not running` ; echo Workers: `ps ax | grep Worker | grep -v grep | wc -l`'",
-		"build": "docker-compose build",
-		"config:core": "rm .env; cp docker-compose-core.yml docker-compose.yml ; npm run build",
-		"config:dev": "echo \"UID=$(id -u)\nGID=$(id -g)\" > .env; docker-compose -f docker-compose-core.yml -f docker-compose-dev-override.yml config > docker-compose.yml; npm run build",
-		"docker:install-chromium-binary": "docker-compose run mshots bash -c 'cd node_modules/puppeteer; node install.js'",
-		"prepare": "npm run config:dev; npm run build; npm run docker:install-chromium-binary"
-	},
-	"dependencies": {
-		"ip": "^1.1.5",
-		"log4js": ">=0.6.0",
-		"puppeteer": "^10.2"
-	},
-	"devDependencies": {
-		"express": "^4.17.1",
-		"jest": "^25.3.0",
-		"sharp": "^0.25.2",
-		"tmp": "^0.1.0"
-	}
+  "name": "mShots.NodeJS",
+  "author": "David Newman <dnewman@automattic.com>",
+  "description": "WordPress mShots System",
+  "version": "0.1.0",
+  "main": "./lib/mshots.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/automattic/mshots.git"
+  },
+  "engines": {
+    "node": ">= 12.0"
+  },
+  "scripts": {
+    "test": "npm run test:php && npm run test:js",
+    "test:php": "phpunit -c tests/phpunit.xml tests",
+    "test:js": "jest",
+    "start": "npm run start:docker && npm run start:mshots && echo mshots service successfully in the mshots docker container. && echo Port 8000 is mapped to localhost, e.g. http://localhost:8000/mshots/v1/example.com",
+    "start:docker": "docker-compose up -d",
+    "stop:docker": "docker-compose down",
+    "start:mshots": "docker-compose exec mshots ./mshots_ctl.sh start",
+    "stop": "docker-compose exec mshots sh -c 'test -f /var/run/mshots.pid && kill `cat /var/run/mshots.pid` && echo Master Process killed && rm -f /var/run/mshots.pid || echo already stopped'; npm run stop:docker",
+    "restart": "npm run stop; npm run start",
+    "status": "docker-compose exec mshots bash -c 'echo mShots Master process: `set -o pipefail; ps ax | grep Master | grep -v grep | awk \"{print \\\\$1}\" || echo not running` ; echo Workers: `ps ax | grep Worker | grep -v grep | wc -l`'",
+    "build": "docker-compose build",
+    "config:core": "rm .env; cp docker-compose-core.yml docker-compose.yml ; npm run build",
+    "config:dev": "echo \"UID=$(id -u)\nGID=$(id -g)\nARCH=$(arch)\" > .env; npm run config:dev:generate-docker-compose; npm run build",
+    "config:dev:generate-docker-compose": "sed 's/google-chrome-unstable/chromium/' Dockerfile > Dockerfile.arm64; docker-compose -f docker-compose-core.yml -f docker-compose-dev-override.yml $(a=$(arch); if [ $a = 'arm64' ]; then echo '-f docker-compose-override-arm64.yml'; fi) config > docker-compose.yml",
```

Comment:
This is the only "magic"
- changes `google-chrome-unstable` to `chromium` using sed, and makes a new Dockerfile.arm64. This is because google-chrome-unstable does not yet have a build for darwin-arm64.
- that is used in the new docker-compose-override-arm64.yaml which is applied only when `arch` = `arm64`